support-frontend/assets/components/productPage/productPagePlanForm/productPagePlanForm.jsx

```diff
@@ -39,24 +41,39 @@ export default function ProductPagePlanForm<P:string>({
     <div
       className="component-product-page-plan-form"
     >
-      <div className="component-product-page-plan-form__items">
+      <div className={`component-product-page-plan-form__items--${theme}`}>
         {keys.map((key: P) => {
             const {
               copy, title, offer, price, saving, href, onClick,
             } = plans[key];
             return (
-              <div className="component-product-page-plan-form__item">
-                <ProductPagePlanFormLabel
-                  {...{
-                  title, offer, key, href, onClick,
-                  }}
-                  footer={((price || saving) ?
-                    <ProductPagePlanFormPrice title={price} copy={saving} />
-                    : null
-                  )}
-                >
-                  {copy}
-                </ProductPagePlanFormLabel>
+              <div className={`component-product-page-plan-form__item--${theme}`}>
+                {theme === 'light' && (
+                  <ProductPagePlanFormLabel
```

Comment:
`ProductPagePlanFormLabel` (light theme) and `ProductPagePlanFormLabelDark` (dark theme) seem very similar, they take the same `props` and `children`, also their `jsx` markup and styles do not look different - other than the use of a darker colour palette for in `ProductPagePlanFormLabelDark` styles. It feels like there's an opportunity to remove the duplication here and have a single component where you pass the theme as a prop and just apply a modifier `--dark` or `--light` modifier to the component's elements' classNames depending on the value of the theme?

Reply:
Yes, on the one hand you are totally correct about the duplication. I guess my thinking was that by keeping them separate, it's easier to get rid of the old (dark) theme when it is phased out, which I believe it will be, rather than having a snaggled mess of css.

Reply:
Ah I see! I didn't realise removing the dark theme was on the horizon, ok in that case it's a sensible decision 👍

Reply:
However, I am checking this assumption with Cloe before merging based on this assumption!

Reply:
Cloe says that is correct 👍